### DISCUSSION TITLE
Authentication error

### DISCUSSION THREAD

**Discussion by jcouzy** [milo526/homebridge-tuya-web#244] [General]
Hi,

I am getting an authentication error, although my credentials are correct.
How can I fix this?

Thanks.

**Comment by milo526** [comment]
Please search through the discussions and issues, many likewise questions have been asked before.

Don't use any federated login,
Don't use special characters in your email-adres.
Try creating a new account if all else fails.

**Reply by beereng** [reply]
Same here since yesterday. Simple and verificated email login, working ok with Tuya app on IOS, don't work here. There's no way to create another fresh account at Tuya's site because I never get a verification code sent by mail. Tried it three times now, no answer. My normal login gets an "invalid" message.

**Reply by milo526** [reply]
The authentication that this plugin is forced to use is rather weird.
I do not see any problems on my side so it does not seem like Tuya changed something.

Did you look at the many other topics regarding this issue and see if they contained something useful?
There is also the [authentication error wiki page](https://github.com/milo526/homebridge-tuya-web/wiki/Authentication-Error) that might help you.

**Reply by jcouzy** [reply]
Hi,

I created a new account with a different email address and suddenly it al seems to work well.
Weird that it works this way, I am very curious why my initially created account doesn't work.
I'll send Tuya/SmartLife an email what has happened at their side.

Thanks!

**Reply by beereng** [reply]
My problem is that I tried to create a new account @Tuya, but never get any email for verification from them, even after one whole day. Now I'm stuck with my old account...

**Reply by milo526** [reply]
I'm sorry to hear that but since I am not affiliated with Tuya, there is nothing I can do to help you in this regard.

**Reply by beereng** [reply]
And Tuya support refuses to answer to my questions regarding my account login, because I'm not a developer or make business with them. F.ck.

**Reply by marcham93** [reply]
Hey, all. I wanted to share my solution for this issue. I had been using the Smart Life app and could not get this plugin to work; it would always say the incorrect username/password was used. The fix for me was to uninstall the Smart Life app and instead install the Tuya app. From within the app, I created a new account and the plugin then worked without issue. I, of course, had to switch all my devices over to the new app though.

**Comment by jcouzy** [comment]
I finally found a solution.

I got rid of the Tuya Smart App en started using the Smart Life App.
I registered a new account for Smart Life and used these credentials within the Tuya Smart Web plugin and selected the Smart Life Platform. Everything works fine now!
